src/sqlancer/tidb/TiDBBugs.java

```diff
@@ -4,46 +4,58 @@
 public final class TiDBBugs {
 
     // https://github.com/pingcap/tidb/issues/15987
-    public static boolean bug15987 = true;
+    public static boolean bug15987;
 
     // // https://github.com/pingcap/tidb/issues/15988
-    public static boolean bug15988 = true;
+    public static boolean bug15988;
 
     // https://github.com/pingcap/tidb/issues/16028
-    public static boolean bug16028 = true;
+    public static boolean bug16028;
 
     // https://github.com/pingcap/tidb/issues/16020
-    public static boolean bug16020 = true;
+    public static boolean bug16020;
```

Comment:
Maybe we can leave them as they are? 
1) It can provide reference information when we find new bugs.
2) Many code locations need to be updated if we remove them.